clang/lib/Sema/SemaOpenMP.cpp

```diff
@@ -21124,6 +21124,8 @@ Sema::ActOnOpenMPDependClause(const OMPDependClause::DependDataTy &Data,
               ExprTy = ATy->getElementType();
             else
               ExprTy = BaseType->getPointeeType();
+            if (ExprTy.isNull())
+              continue;
```

Comment:
I'm not sure to `continue` here is a good idea. Probably you want `return nullptr;`.

Reply:
done.